src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_MediaPlayerElement.cs

```diff
@@ -11,38 +13,247 @@
 
 using _MediaPlayer = Windows.Media.Playback.MediaPlayer; // alias to avoid same name root namespace from ios/macos
 
-namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls;
+
+[TestClass]
+[RunsOnUIThread]
+public partial class Given_MediaPlayerElement
 {
-	[TestClass]
-	[RunsOnUIThread]
-	public partial class Given_MediaPlayerElement
+	private static readonly Uri TestVideoUrl = new Uri("https://test-videos.co.uk/vids/bigbuckbunny/mp4/h264/720/Big_Buck_Bunny_720_10s_5MB.mp4");
+
+	[TestMethod]
+	public async void When_NotAutoPlay_Source()
```

Comment:
```suggestion
	public async Task When_NotAutoPlay_Source()
```
convert to `async Task` for all

Reply:
Xiao, using task the CI is not processing the tests. It is waiting for the MPE to be loaded and does not execute.
I had to take them all out and use void.
I believe the CI is running one at a time. But, I didn't find it or I don't have access to confirm.

Reply:
@Xiaoy312 , can i close this comment? with th Task the CI do not validate.

Reply:
i put Task back. Thanks.